src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+    public Circle(String color , double radius) {
```

Comment:
There is an extra space before the comma in the constructor parameter list `String color , double radius`. It should be `String color, double radius`.